projects/plugins/jetpack/class.jetpack.php

```diff
@@ -6552,10 +6555,9 @@ public static function is_rewind_enabled() {
 			$rewind_data    = (array) Jetpack_Core_Json_Api_Endpoints::rewind_data();
 			$rewind_enabled = ( ! is_wp_error( $rewind_data )
 				&& ! empty( $rewind_data['state'] )
-				&& 'active' === $rewind_data['state'] )
+				&& 'unavailable' !== $rewind_data['state'] )
```

Comment:
Backup state can be "awaiting credentials" so I switched the conditional to using "unavailable" instead.